README.md

```diff
@@ -24,21 +24,21 @@ registry=https://nexus.mntd.net/repository/npm-minted-group/
 ```
 $ nvm use
 $ yarn install
+$ yarn build-server
 $ yarn dev
```

Comment:
I don't think we need the `yarn build-server` command.

Reply:
currently we are requiring in the build server.js bundle at https://github.com/mwang102/minted-frontend-boilerplate/blob/a0fd057ebdc2e99a762a0b3fdacdd41f06a7b595/src/server.js#L27 . The app-r build bundle command is `yarn build-app-renderer-server` not `yarn build-server`

Reply:
Yes, you're requiring it there, but you're also building the server bundle each time a change is made when running `yarn dev`:
https://github.com/mwang102/minted-frontend-boilerplate/blob/a0fd057ebdc2e99a762a0b3fdacdd41f06a7b595/src/server.js#L11

If we didn't do this, you'd get React errors constantly about an issue with the server-side not matching the client-side render when hydrating.

Reply:
hmm interesting. you're correct